examples/flexvolume/nginx.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+    volumeMounts:
+    - name: test
+      mountPath: /data
+    ports:
+    - containerPort: 80
+  volumes:
+  - name: test
+    flexVolume:
+      driver: "kubernetes.io/lvm"
+      fsType: "ext4"
+      options:
+        volumeID: "vol1"
+        size: "1000m"
+        volumegroup: "kube_vg"
+        fsType: "ext4"
```

Comment:
Darn, I just noticed that. I know we're near the finish line, but could you split the mount unmount into a separate callouts, and default to the default in-binary `FormatAndMount` code if the mount/unmount callouts are not provided.